src/model/course.js

```diff
@@ -0,0 +1,45 @@
+import mongoose from 'mongoose';
+
+const courseSchema = new mongoose.Schema(

```

Comment:
probably you have to add another entry and name it `creator` and pass User schema as ref. see similar pattern in  https://github.com/yusuf963/jobtal-api/blob/main/src/model/jobPost.js#L18